### ISSUE TITLE
Error page did not disappear

### ISSUE THREAD

**Issue by henning-gerhardt** [kitodo/kitodo-production#4026]
I would to start newspaper migration or better call only the migration page, an error appear but then I can not reopen any "system" page as I get every time an error page:

![Screenshot_2020-10-07_12-04-02](https://user-images.githubusercontent.com/2129672/95317264-35c22f80-0895-11eb-8267-3ce334bf402c.png)

Re-logging did not help, restarting Tomcat service is only an option in non-productive environments.

used version: master branch 2020-10-05 / commit 6c5b92a674159520c1929549b80e44b5568be99f

**Comment by Kathrin-Huber** [comment]
Any errors in console?
The ViewExpiredException is usally just a following error to another one.

**Comment by henning-gerhardt** [comment]
Main error event is described in #4027 After the described error was shown, only this error is still visible. There are thousand of lines in the log file of Kitodo.Production and Tomcat (catalina.out), so I can only post the first few lines:

```
":"too_many_clauses","reason":"maxClauseCount is set to 1024"}}}]},"status":400}
                        at org.elasticsearch.client.RestClient$1.completed(RestClient.java:354)
                        at org.elasticsearch.client.RestClient$1.completed(RestClient.java:343)
                        at org.apache.http.concurrent.BasicFuture.completed(BasicFuture.java:119)
                        at org.apache.http.impl.nio.client.DefaultClientExchangeHandlerImpl.responseCompleted(DefaultClientExchangeHandlerImpl.java:177)
                        at org.apache.http.nio.protocol.HttpAsyncRequestExecutor.processResponse(HttpAsyncRequestExecutor.java:436)
                        at org.apache.http.nio.protocol.HttpAsyncRequestExecutor.inputReady(HttpAsyncRequestExecutor.java:326)
                        at org.apache.http.impl.nio.DefaultNHttpClientConnection.consumeInput(DefaultNHttpClientConnection.java:265)
                        at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:81)
                        at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:39)
                        at org.apache.http.impl.nio.reactor.AbstractIODispatch.inputReady(AbstractIODispatch.java:114)
                        at org.apache.http.impl.nio.reactor.BaseIOReactor.readable(BaseIOReactor.java:162)
                        at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvent(AbstractIOReactor.java:337)
                        at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvents(AbstractIOReactor.java:315)
                        at org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:276)
                        at org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:104)
                        at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:588)
                        ... 1 more
07-Oct-2020 11:42:01.885 SEVERE [http-nio-127.0.0.1-8080-exec-2] org.apache.myfaces.context.servlet.ServletExternalContextImpl.setResponseContentType Cannot set content type. Response already committed
07-Oct-2020 11:42:02.184 SEVERE [http-nio-127.0.0.1-8080-exec-4] org.omnifaces.exceptionhandler.FullAjaxExceptionHandler.logException FullAjaxExceptionHandler: An exception occurred during processing JSF ajax request. Error page '/pages/error.jsf' will be shown.
        javax.faces.application.ViewExpiredException: View "/pages/system.jsf" could not be restored.
                at org.apache.myfaces.lifecycle.RestoreViewExecutor.execute(RestoreViewExecutor.java:181)
                at org.apache.myfaces.lifecycle.LifecycleImpl.executePhase(LifecycleImpl.java:195)
                at org.apache.myfaces.lifecycle.LifecycleImpl.execute(LifecycleImpl.java:142)
                at javax.faces.webapp.FacesServlet.service(FacesServlet.java:204)
                at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
                at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
                at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
                at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
                at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
                at org.kitodo.production.servletfilter.EncodingFilter.doFilter(EncodingFilter.java:69)
                at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
                at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
                at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:317)
                at org.kitodo.production.security.SecurityObjectAccessFilter.doFilter(SecurityObjectAccessFilter.java:86)
                at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
                at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.invoke(FilterSecurityInterceptor.java:127)
                at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.doFilter(FilterSecurityInterceptor.java:91)
                at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
...
```

**Comment by matthias-ronge** [comment]
Was this fixed by  #4055?

**Comment by henning-gerhardt** [comment]
I think so. At least I can not reproduce the error condition any  more.
